### PR TITLE
updates for tensorflow 1.0

### PR DESCRIPTION
**Pull request by dribnet**
Hi - great project. I attempted to update it to work with tensorflow 1.0. With these code changes, it trains a model and samples correctly, but so far (training progress: 50000/125000) I'm not getting coherent samples from the model - as can be seen in the [sample notebook](https://github.com/dribnet/scribe/blob/877aabd639a096c0e9ac34bec1b15627209dabff/sample.ipynb). So there might be something subtle still not right, but I'll go ahead and post this in case there is interest in updating the project.

### CODE REVIEW

**Comment by greydanus** [comment]
@dribnet The training process is tricky; I had to try quite a range of hyperparameters before I got good results. I agree that there might be something subtle that's causing an issue; it could be in the loss or in the mixture density portion of the code. By 50000 you should be getting semi-coherent results

**Comment by dribnet** [comment]
Thanks for the feedback. I was able to track down the bug - it's actually failing to load the saved model when run from `sample.ipynb` - the message is `Key model/mdn_dense/output_b not found in checkpoint`.  I've confirmed this behavior in both this branch (with tensorflow 1.0) and the current master branch (with tensorflow 0.12.0) , so this seems to be something unrelated to my changes.

But if I instead sample the model using `run.py`, I get sensible results (attached), though the conditioning partially failing to match the input string.  I'm going to try training with a few other hyperparameters and sampling with `run.py` to see if I can get a model that more closely matches your results.

iter-124500-l-you_know_n

iter-124500-g-you_know_n

iter-124500-w-you_know_n

**Comment by greydanus** [comment]
Looks like you're updates are useful! Merging
